lib/linguist/languages.yml

```
@@ -1113,6 +1121,13 @@ Cuda:
   codemirror_mime_type: text/x-c++src
   color: "#3A4E3A"
   language_id: 77
+Cue sheet:
```

Titlecase is preferred: 

```suggestion
Cue Sheet:
```